src/app/core/services/authentication.service.ts

```diff
@@ -57,6 +58,34 @@ export class AuthenticationService {
     return from(this.auth.authState);
   }
 
+  updateDisplayName(name: string) {
+    this.auth.authState.pipe(take(1)).subscribe((user) => {
+      if (user) user.updateProfile({ displayName: name });
+    });
+  }
+
+  updatePassword(password: string) {
+    this.auth.authState.pipe(take(1)).subscribe((user) => {
+      if (user) user.updatePassword(password);
```

Comment:
My opinion: Never omit brackets. Error prone. There should be a linting rule to forbid it to be honest 😬